package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Gollums Game",
```

Comment:
There are restrictions on the name property.  It has to be one word with no spaces and no capital letters.  Otherwise npm will throw an error and not load your packages.